lib/Resource.js

```diff
@@ -37,9 +37,10 @@ export default class Resource {
 
   _responseHandler (res) {
     const response = clone(res.data)
+    const headers = res.headers || {}
 
     const lastResponse = {
-      requestId: res.headers['x-request-id'],
+      requestId: headers['x-request-id'],
```

Comment:
Just curious to know if there was any reason to make this small refactoring change in this PR. Anyway it’s probably OK.

Reply:
Priorly Stelace SDK is expected to hit Stelace API, which always provides the header `x-request-id`.

However by introducing the forward methods in this PR, custom endpoints can not provide this header. I found it by stubbing a response without this header in tests.
That's why I didn't isolate this portion of code into a separate commit.